frontend/src/v5/ui/routes/dashboard/projects/tickets/ticketsTable/ticketsTableContent/ticketsTableGroup/ticketsTableGroup.component.tsx

```diff
@@ -126,6 +131,7 @@ export const TicketsTableGroup = ({ tickets, onEditTicket, onNewTicket, selected
 										selected={selectedTicketId === ticket._id}
 									/>
 								))}
+								{!template.deprecated &&
```

Comment:
I was looking at a template from the tabular view, then I set that same template to "deprecated", and refreshed. The app crashes with this error:
![image](https://github.com/user-attachments/assets/ded051c8-aa5d-46a0-b1dd-cbdfac2a02b6)

As such, I think this is not necessary as a deprecated template is not reacheable, but we may need a way to handle invalid templates passed in the URL

Reply:
Cant reproduce this with a deprecated template. Yes with an  invalid templatem but that is out of scope.